src/app.ts

```diff
@@ -0,0 +1,46 @@
+import { createServer, IncomingMessage, ServerResponse } from 'http';
+import { config } from './config';
+import { logger } from './logger';
+import { noAppPortMsg, noEnvMsg } from './constants';
+import GetCommand from './commands/getCommand';
+import PostCommand from './commands/postCommand';
+
+if (config.APP_PORT === undefined) {
+    logger.info(noAppPortMsg);
+    throw new Error(noAppPortMsg);
+}
+  
+if (config.ENV === undefined){
+    logger.info(noEnvMsg);
+    throw new Error(noEnvMsg);
+}
+
+const contentTypeHeader = { 'Content-Type': 'application/json' };
+const message = `Listening on port: ${config.APP_PORT}. ENV is ${config.ENV}`;
+
+createServer((request: IncomingMessage, response: ServerResponse) => {
+    logger.info('New incoming request');
+    logger.log('Request Type', request.method);
+
+    response.writeHead(200, contentTypeHeader);
+    switch(request.method){
+        case 'GET':
+            new GetCommand(request, response).run();
```

Comment:
You don't need classes for request handlers. In fact, you are creating a new instance of `GetCommand` for every server call, that's no good. Better use functions there.